src/features/CheckoutPage/components/PaymentInformation.jsx

```diff
@@ -0,0 +1,93 @@
+
+import React, {useState, Fragment} from 'react';
+import {Listbox} from '@headlessui/react';
+import {AiOutlineCheck} from 'react-icons/ai';
+import PropTypes from 'prop-types';
+import {MONTHS} from '../../../constants';
+
+const PaymentInformation = ({submitHandler}) => {
+  const [name, setName] = useState('');
+  const [number, setNumber] = useState('');
+  const [month, setMonth] = useState('January');
+  const [year, setYear] = useState('2025');
+  const [cvv, setCVV] = useState('');
+  const formActionMappings = {
+    'name': setName,
+    'number': setNumber,
+    'month': setMonth,
+    'year': setYear,
+    'cvv': setCVV,
+  };
+
+  const onFormInputChange = async (e) => {
```

Comment:
all optional chaining for nullish coalescence

Reply:
done